Formula/ldc.rb

```diff
@@ -23,7 +23,7 @@ class Ldc < Formula
   depends_on "cmake" => :build
   depends_on "libconfig" => :build
   depends_on "pkg-config" => :build
-  depends_on "llvm@12"
+  depends_on "llvm"
 
   uses_from_macos "libxml2" => :build
   # CompilerSelectionError: ldc cannot be built with any available compilers.
```

Comment:
Should hopefully be able to remove this and the following line now if we no longer rely on `llvm@12` (which Homebrew doesn't consider a valid compiler for selection)

Reply:
I've dropped those lines and everything worked great, so they are no longer needed.